templates/play.html

```diff
@@ -26,11 +26,48 @@
         <audio id="music2" src="/static/bgm/bgm.mp3"  loop="loop">你的瀏覽器不支援audio標籤。</audio>    
         <a href="javascript:playPause();"><img src="{{url_for('static',filename='/images/pause.png')}}" width="35" height="35" id="music_btn2" ></a>
       </div>
+      <div>
+        <div class>
+          <div class="lightning">
+            <div class="lightning1">
+              <img class="crack1" src="{{url_for('static',filename='/images/redlight.png')}}" height="400" width="400">
+            </div>
+            <div class="lightning2">
+              <img class="crack2" src="{{url_for('static',filename='/images/greenlight.png')}}" height="400" width="400">
+            </div>
+            <div class="lightning3">
+              <img class="crack3" src="{{url_for('static',filename='/images/purplelight.png')}}" height="400" width="400">
+            </div>
+            <div class="lightning4">
+              <img class="crack4" src="{{url_for('static',filename='/images/bluelight.png')}}" height="400" width="400">
+            </div>
+          </div>
+        </div>
+      </div>
       <div class="sky text container">
         <div class="row">
+          <div class = "col"></div>
+          <div class = "col-1" >
+            <img id="heart" src="{{url_for('static', filename='/images/heart.png')}}" widht="80" height="80" id="heart" onclick="changeImage()">
+          </div>
+          <div class = "col-1">
+            <img id="heart1" src="{{url_for('static', filename='/images/heart.png')}}" widht="80" height="80" id="heart1" onclick="changeImage1()">
+          </div>
+          <div class = "col-1">
+            <img id="heart2" src="{{url_for('static', filename='/images/heart.png')}}" widht="80" height="80" id="heart2" onclick="changeImage2()">
+          </div>
+          <div class = "col-1">
+            <img id="heart3" src="{{url_for('static', filename='/images/heart.png')}}" widht="80" height="80" id="heart2" onclick="changeImage3()">
+          </div>
+          <div class = "col-1">
+            <img id="heart4" src="{{url_for('static', filename='/images/heart.png')}}" widht="80" height="80" id="heart2" onclick="changeImage4()">
+          </div>
```

Comment:
Too many heart id `heart2`.

Rewrite this by jinja2 templates for-loop to avoid the same code.

Reply:
Ok, I will change it

Reply:
done

Reply:
Commit and push it to the repo.